vividus-plugin-web-app/src/main/java/org/vividus/bdd/steps/ui/web/ActionSteps.java

```diff
@@ -175,6 +176,8 @@ public void executeActionsSequence(List<Action> actions)
      * <td>PRESS_KEYS</td>
      * <td><a href="https://selenium.dev/selenium/docs/api/java/org/openqa/selenium/Keys.html">Keys</a>
      * </td>
+     * <td>PRESS_KEYS_SIMULTANEOUSLY</td>
+     * <td>press keys combination such as "ctrl + a" etc.</td>
```

Comment:
can I press more than two keys?

Reply:
If you mean pressing more than one key simultaneously (Example "ctrl + alt + delete") I think no, you can't. For this issue, I've seen another solution than every key that you want to press simultaneously should be realized by the "keyDown()" method.

Reply:
rewrite